integration/tests/cook/test_basic.py

```diff
@@ -230,7 +230,9 @@ def test_docker_fields(self):
         self.assertEqual(docker_image, docker['image'])
         self.assertEqual('HOST', docker['network'])
         self.assertEqual(False, docker['force-pull-image'])
-        self.assertEqual(2, len(docker['parameters']))
+        # the user parameter is added when missing
+        self.assertEqual(3, len(docker['parameters']))
+        self.assertTrue(next(True for p in docker['parameters'] if p['key'] == 'env'))
```

Comment:
Should this be:

```suggestion
        self.assertTrue(any(p['key'] == 'user' for p in docker['parameters']))
```

(using `any` and also testing for the `'user'` key?)